src/schema.ts

```diff
@@ -407,6 +407,7 @@ const PhoneCodeSchema = new Schema({
     type: Number,
     required: true,
   },
+  sms_provider: String,
```

Comment:
I think `sms_` might be obvious/implied given the collection. `provider` would probably be enough?